### ISSUE TITLE
Bug: loadGlBible() in translation helps is loading bibles from Door43-Catalog - should be reading from current org

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/gateway-edit#417]

**Comment by theNerd247** [comment]
I think this issue, technically, should belong in [translation-helps-rcl](https://github.com/unfoldingWord/translation-helps-rcl) since that's where the code changes will be made. @PhotoNomad0 @birchamp do we have a mechanism to link this issue to a copy in the THR repo?

**Comment by PhotoNomad0** [comment]
> I think this issue, technically, should belong in [translation-helps-rcl](https://github.com/unfoldingWord/translation-helps-rcl) since that's where the code changes will be made. @PhotoNomad0 @birchamp do we have a mechanism to link this issue to a copy in the THR repo?

Correct, I was trying to communicate in the issue description that the fix needs to be made in the translation-helps-rcl.  When you create a PR for the fix you will need to connect the PR to this issue using zenhub. FWI - If you have prettier turned on by default, please turn if off before making edits.  Otherwise it becomes a major pain reviewing PRs.

**Comment by theNerd247** [comment]
Sounds good! I’m not a fan of formatters anyway so that shouldn’t be a problem 😁
